### PR TITLE
Update product description help text in CMS

### LINKED ISSUE #179: Update help text for description field for course page in CMS
### Steps to Reproduce

- Edit a course page in the CMS: e.g. https://rc.mitxonline.mit.edu/cms/pages/8/edit/
- Roll over the description section
- Note the help text on the right says `The description shown on the product page`

### Expected Behavior

- It should ideally say `The description shown on the home page and product page.`

### Code
- I believe the text is specified [here](https://github.com/mitodl/mitxonline/blob/5a5454f6cb5926e4a0ed7a0baaf6e9460d926410/cms/models.py#L212)

### PR DESCRIPTION
**Pull request by arslanashraf7**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
- [x] Migrations
  - [x] Migration is backwards-compatible with current production code
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Closes #179 

#### What's this PR do?
- Updates the help text for the description field in the product/course model

#### How should this be manually tested?
- Visit any existing course page or create a new one under the courses page
- Hover over the description field and you should see an updated description message (See screenshot for ref) 

#### Screenshots (if appropriate)
Screenshot 2021-09-17 at 5 25 42 PM

### CODE REVIEW

**Comment by arslanashraf7** [comment]
@briangrossman Can you take a look at the screenshot in the description and let me know if there are any concerns.

**Comment by briangrossman** [comment]
> @briangrossman Can you take a look at the screenshot in the description and let me know if there are any concerns.

@arslanashraf7 : Screen shot looks good to me 👍

File: cms/migrations/0012_product_description_help_text.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    dependencies = [
+        ('cms', '0011_product_faculty_members_optional'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='coursepage',
+            name='description',
+            field=wagtail.core.fields.RichTextField(blank=True, help_text='The description shown on the home page and product page.'),
```
Comment (umarmughal824) [inline cms/migrations/0012_product_description_help_text.py:17]: Just run the `black` command so that it could follow pep8 line length restrictions.

**Review by umarmughal824 — APPROVED** [review]
Looks good 👍 , just address one minor concerns.

**Pull request merged**